docs/DEPLOYMENT.md

```diff
@@ -122,23 +126,26 @@ db-migration-8vhpd     0/1     Completed   0          17s
 
 You can also check the logs using `kubectl logs -n tekton-hub db-migration-8vhpd`
 
-The migration logs at the end should show
+The status of `Pod` would be `Error` if the job fails due to any reason
 
 ```bash
-2020-09-22T15:35:16.412Z INFO migration/migration.go:91 Migration ran successfully !! {"service": "migration"}
-2020-09-22T15:35:16.412Z INFO db/main.go:39 DB initialisation successful !! {"service": "main"}
+$ kubectl get pods -n tekton-hub
+NAME                   READY   STATUS      RESTARTS   AGE
+db-589d44fdd5-ksf8v    1/1     Running     0          50s
+db-migration-8vhpd     0/1     Error       0          17s
```

Comment:
So this is the status of the pod when there is an error, but since this is the deployment docs I think we should add the status of the pod when it gets succeeded, wdyt ??

Reply:
I guess it's already there. Ref: https://github.com/tektoncd/hub/pull/318/files#diff-ea9864292e6e392d0c8edbabfe41014d61d6ebd2e08f34d8ae21bf9d070fb274L120